src/Traits/TenantAwareCommand.php

```diff
@@ -26,6 +26,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->laravel->call([$this, 'handle']);
             });
         }
+
+        return 1;
```

Comment:
@devonmather Actually, why are we returning `1` here? What's the reason for this and why `1` and not `0`?

Reply:
The reason is it must return an integer value, I think it's related to the latest symfony component for commands. Upon further reading just now. I misunderstood the error and figured a 0 would be if the command failed. But upon further reading it looks like the default return value is zero. I can created another PR this evening to fix.

Reply:
@devonmather Seems like Laravel casts the return value from `handle()` into an int and returns that. See this: https://github.com/laravel/framework/blob/7.x/src/Illuminate/Console/Command.php#L134

I guess most `handle()` methods return nothing, so null, which is cast into `0` - as success? And then the `handle()` method can optionally return for example `1` as a specific exit code? I'll open a PR and ask you for review, if you'd be so kind. I don't have that much experience with commands.